<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserForm</class>
 <widget class="QWidget" name="UserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paner assault</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: &quot;Yu Gothic&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="titleLine">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/tank.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="title">
       <property name="styleSheet">
        <string notr="true">font-size: 20pt;</string>
       </property>
       <property name="text">
        <string>Panzer assault</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="serverInfoLine">
     <item>
      <widget class="QPushButton" name="buttonConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Status: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusNow">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt; color: green;</string>
       </property>
       <property name="text">
        <string>online</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ping">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>ping to server:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pingMs">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>0 ms</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLine">
     <item>
      <widget class="QPushButton" name="buttonBattle">
       <property name="text">
        <string>Battle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonResearch">
       <property name="text">
        <string>Research</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonShop">
       <property name="text">
        <string>Shop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="accountInfo">
     <item>
      <layout class="QFormLayout" name="infoForm">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="credits">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Credits:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="creditsAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="experience">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Experience:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="experienceAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="bonuses">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Bonuses:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="bonucesAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="timeUser">
       <property name="styleSheet">
        <string notr="true">font-size: 15pt</string>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="machineInfo">
     <item>
      <widget class="QLabel" name="machine">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>Selected machine:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="machineName">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPrev">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="machinePreview">
     <item>
      <layout class="QFormLayout" name="machineCharacteristics">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="health">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Health:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="healthAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="speed">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="speedAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="rotateSpeed">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Rotate speed:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="rotateSpeedAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="damage">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Damage:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="damageAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="attackSpeed">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Cooldown time:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="attackSpeedAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="armor">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Armor:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="armorAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="bulletSpeed">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Bullet speed:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="bulletSpeedAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="ammoCapacity">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>Ammo capacity:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="ammoCapacityAmong">
         <property name="styleSheet">
          <string notr="true">font-size: 9pt;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="machineSkin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>175</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
